cmd/zq/zq.go

```diff
@@ -161,6 +162,18 @@ func isTerminal(f *os.File) bool {
 }
 
 func (c *Command) Run(args []string) error {
+	if true {
```

Comment:
Debug relic. Were you want to add a flag to zq for this?

Reply:
whoops, didn't mean to include this on the branch, will remove